src/VolleyManagement.API/Properties/launchSettings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:49940/",
+      "sslPort": 44370
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "launchUrl": "api/Tournaments",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "VolleyManagement.API": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "api/Tournaments",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "http://localhost:49941/"
```

Comment:
Match port

Reply:
These are different launch settings. You can choose them by navigating: Project properties > Debug > Launch > select from dropdown.

Reply:
The launchSettings.json file controls what environments you can run your app in using Visual Studio. This means that we can run the app in multiple different environment settings from Visual Studio, by setting up the appropriate launch profiles.